sdk/eventhubs/azure_messaging_eventhubs/tests/checkpoint_store.rs

```diff
@@ -0,0 +1,160 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+//! # Event Hubs SDK for Rust
+//!
+//!
+
+mod common;
+use azure_core::error::ErrorKind as AzureErrorKind;
+use azure_messaging_eventhubs::CheckpointStore;
+use std::sync::Arc;
+
+#[cfg(test)]
+mod tests {
```

Comment:
Don't put tests in a `tests` module when already under `tests/`. Did you check if these even ran? I'd be surprised. Even if they did, it's redundant.